cmd/agent/app/integrations.go

```diff
@@ -257,19 +301,15 @@ func pip(args []string, stdout io.Writer, stderr io.Writer) error {
 }
 
 func install(cmd *cobra.Command, args []string) error {
+	detectEnvironment()
```

Comment:
As mentioned earlier, this should possibly return an error if we can't detect the environment reliably and exit.